src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -87,7 +87,12 @@ public function admin_init() {
 	 * @return string success message
 	 */
 	public function success_notice() {
-		return '<div class="success"><p>' . __( 'Attendee Registration fields saved.', 'event-tickets' ) . '</p></div>';
+		$link    = '<a href="' . get_edit_post_link( $this->post->ID, 'raw' ) . '">' . __( 'return to the content editor', 'event-tickets' ) . '</a>';
```

Comment:
Should have escaping on the link and text

```suggestion
		$link    = '<a href="' . esc_url( get_edit_post_link( $this->post->ID, 'raw' ) ) . '">' . esc_html__( 'return to the content editor', 'event-tickets' ) . '</a>';
```